backend/ttrs/views.py

```diff
@@ -150,3 +150,13 @@ class MajorDetail(generics.RetrieveAPIView):
     queryset = Major.objects.all()
     serializer_class = MajorSerializer
     permission_classes = (AllowAny,)
+
+from .recommend import recommend
+
+class RecommendView(generics.RetrieveAPIView):
+    permission_classes = (AllowAny,)
+
+    def get(self, request):
+        options = request.query_params.get('options')
```

Comment:
It would be better to make `options` to python dictionary, not plain text so that there is no need to parse JSON text.
It can be true if you uses `request.query_params` as `options`.